app/scripts/modules/core/instance/details/multipleInstances.controller.js

```diff
@@ -94,7 +94,7 @@ module.exports = angular.module('spinnaker.core.instance.details.multipleInstanc
 
     this.canRegisterWithDiscovery = () => allDiscoveryHealthsMatch('OutOfService');
 
-    this.canDeregisterWithDiscovery = () => allDiscoveryHealthsMatch('Up');
+    this.canDeregisterWithDiscovery = () => allDiscoveryHealthsMatch('Up') || allDiscoveryHealthsMatch('Down');
```

Comment:
not super proud of this since it means that if you had selections that were up and down it wouldn't enable the dialog box, but down instances are pretty rare that I think it'll be ok instead of rewriting the logic in allDisvoceryHealthsMatch